pkg/service/manipulator.go

```diff
@@ -17,12 +19,16 @@ const (
 	crop         = "crop"
 	mono         = "mono"
 	blackHexCode = "000000"
+	flip         = "flip"
+	rotate       = "rotate"
```

Comment:
Can you update this to `rotate = "rot"`? It'll make the URL shorter when we support more features.